swim/member_test.go

```diff
@@ -139,3 +94,16 @@ func TestChangeOmitTombstone(t *testing.T) {
 	_, has := parsedMap["tombstone"]
 	assert.False(t, has, "don't expect the tombstone field to be serialized when it is")
 }
+
+func TestMemberChecksumString(t *testing.T) {
+	member := Member{
+		Address:     "192.168.2.1:1234",
+		Status:      Alive,
+		Incarnation: 42,
+	}
+
+	var b bytes.Buffer
+	member.checksumString(&b)
+
+	assert.Equal(t, "192.168.2.1:1234alive42", b.String(), "member checksum serialization failed")
```

Comment:
True, but in retrospect I will remove this test as it is the exact same test as the doc_test above. Doesn't feel right to test the same thing twice.